packages/stark-ui/src/modules/dropdown/components/_dropdown.component.scss

```diff
@@ -6,20 +6,33 @@
   mat-select {
     margin: 0;
   }
-  width: 150px;
+
+  .mat-form-field-infix {
+    width: 150px;
+  }
+
+  &[width="full"] {
```

Comment:
I know we have been working with `color="white"` for the color but it is exceptional cause we want to be aline with Material. In other cases, we should use classes instead, so here it should be:

```
&.stark-full-width{
  .mat-form-field-infix {
    width: 100%;
  }
}
```